nuclear/CMakeLists.txt

```diff
@@ -1,6 +1,3 @@
-# We use additional modules for the NUClear roles system
-list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake/Modules/")
```

Comment:
Pretty sure this should stay here. NUClear roles needs to be able to use the things in NUClear roles.
And no the one in the root cmakelists doesn't count, it's not NUclear roles yet `IWYU`